translations/config.json

```diff
@@ -125,7 +125,8 @@
         "SergeyMosin appointments",
         "stjosh auto_groups",
         "sualko cloud_bbb",
-        "tacruc gpgmailer"
+        "tacruc gpgmailer",
+		"westberliner checksum"
```

Comment:
Please fix indention. Tabs to spaces.

Reply:
Should be fixed. Somehow the browser changed spaces to tabs. Sry for that.